index.js

```diff
@@ -186,7 +188,9 @@ module.exports = {
         directory: result.directory,
         indexFile: this.app.options.outputPaths.app.html,
         sha: indexSha,
+        configPath: this.app.options.amd.configPath,
```

Comment:
The idea of passing an object to the function indexBuilder was to carry the particular properties for one index over the other index (runtime vs test). I wouldn't pass the global config properties configPath and inline. They can be accessed by the indexBuilder function directly.